Gameboard.java

```diff
@@ -15,6 +15,8 @@
  	private static FINAL int BOARD_SIZE = 100;
  	private static FINAL int ROWS = 10;
  	private static FINAL int COLUMNS = 10;
+     
+    //isn't it [ROWS][COLUMNS] for a two-dimensional array?
```

Comment:
Yes, it usually is row columns. However, for battleship specifically the columns are letters and rows are numbers, and when you chose a spot to attack, you always say A-4 or B-10 (so column first, then row)